### ISSUE TITLE
ImageView in Sample mode still loads full resolution content

### ISSUE THREAD

**Issue by SeranaX** [NO-ob/LoliSnatcher_Droid#190]
When selecting the "Sample" option in the Gallery settings, the ImageView should load images with sample resolution instead of the full resolution, isn't it?

**Steps to Reproduce**
1. Swipe to open the menu
2. Go to "Settings" -> "Gallery" -> "Gallery Quality"
3. Select "Sample"
4. Return to the main (grid) page
5. Swipe down to refresh links
6. Open any picture

**Expected behavior**
The ImageView should load sample resolution content in the gallery when the "Sample" option is selected in the Gallery settings.

**Actual behavior**
The ImageView loads the full resolution content in the gallery even when the "Sample" option is selected in the Gallery settings.

**Affects**
This issue affects both the interaction speed and the amount of traffic consumed.

**Comment by SeranaX** [comment]
Most likely the problem is here:
https://github.com/NO-ob/LoliSnatcher_Droid/blob/8b83935bec08264a5da317b434b66c2170c3cebf/lib/src/widgets/image/image_viewer.dart#L200
I propose to replace this line with the following code:
```
(settingsHandler.galleryMode == "Sample")
    ? widget.booruItem.sampleURL
    : widget.booruItem.fileURL,
```

**Comment by NANI-SORE** [comment]
Oops, I accidentally dropped that setting check when reworking image widget
Will fix in the next release

**Comment by NANI-SORE** [comment]
Fixed in 2.3.3
